code/modules/assembly/flash.dm

```diff
@@ -322,6 +323,56 @@
 /obj/item/assembly/flash/cyborg/screwdriver_act(mob/living/user, obj/item/I)
 	return
 
+
+///Camera flash
+
+/obj/item/assembly/flash/camera
+	name = "camera"
+	icon = 'icons/obj/items_and_weapons.dmi'
+	desc = "A polaroid camera."
+	icon_state = "camera"
+	inhand_icon_state = "camera"
+	worn_icon_state = "camera"
+	lefthand_file = 'icons/mob/inhands/misc/devices_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/misc/devices_righthand.dmi'
+	flags_1 = CONDUCT_1
+	slot_flags = ITEM_SLOT_NECK
+	pickup_sound = null
+	flash_sound = 'sound/items/polaroid1.ogg'
+	var/max_charges = 5
+	var/current_charges = 5
+	var/list/charge_timers = list()
+	var/charge_time = 300 //30 seconds
+
+/obj/item/assembly/flash/camera/burn_out()
+	return //use self charging system instead
+
+/obj/item/assembly/flash/camera/attack(mob/living/M, mob/user)
+	if(current_charges)
+		current_charges --
+		to_chat(user, span_notice("You use [src]. It now has [current_charges] charge[current_charges == 1 ? "" : "s"] remaining."))
```

Comment:
The inbuilt `\s` macro seems to work fine with this:
![image](https://user-images.githubusercontent.com/57483089/168159921-bcb563a2-bbe3-4577-8bc7-23417b764588.png)
```suggestion
		to_chat(user, span_notice("You use [src]. It now has [current_charges] charge\s remaining."))
```